sale_analysis_report/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale Analysis Report
+====================
+
+ This module extends the functionality of Project to add a category on the task and support filter and group.
```

Comment:
looks incorrect.
Anyway, I am not sure to understand the logic of this module as it seems a simple copy of standard code.